### PR TITLE
Bombards Split personality with an antigrain warhead

### PR DESCRIPTION
**Pull request by IrkallaEpsilon**
Porting from my old ### **_high effort high performance coding tradition and good coding( T&G coding)_** PR to make people not walk into unpleasant things if somebody rolls split personality during ERP (or in general as this is still a shitter magnet)

Old Description with some minor updates:
Split personality is causing issues as expected. Hijacking a character randomly is iffy at best. At worst we have severe pref breaks e.g. when you get shoved into a catgirl handhold tailtwine infront of a bunch of people that dont like watching youwu twalk like tis while having neon green athletics shorts and long neon green hair ERP when you are neither into catgirls nor into handhold tailtwining nor being into neon colors nor any other things listed here for the given example.
_'Authors comment: I cannot believe I am giving a hypothetical example on this.'_
 It was brought up time and time again yet nobody did the PR to gut this meme from TG.

It is constantly being abused for LRP shittery and the few good split personalities are far and between and not worth the hassle.

Its legit a "har I get to grief this person now" or "Ewwww your kink is gross". Nothing more. Nothing less.

Why It's Good For The Game
🦀 Split personality is gone 🦀

Its easier to list the reasons on why split personaliy is bad for the game. Gist of it: Its a nightmare to handle for administration, a massive saltmine for players, Grief and LRP autocatalyst (meaning it creates situations which in term create more situations for LRP, the catalyst is producing more catalyst).

For those defending split personality: No I wasnt recently affected by that (I havent played in months), couple other people filing complains however were.
So why not get rid of it altogether.

Also then theres this entire stick of being thrown into somebodies ERP as already mentioned and one not necessarily wanting to see certain things.

DISCLAIMER:

There is absolutely nothing wrong with catgirls, handholding, tailtwining, neon colors or whatever I listed above. The examples used were simply chosen since the combination of these in one sentence have a memorable effect to some capacity.

🦀 Split personality is gone 🦀

Changelog
🆑
del: Thanks to people breaking prefs before rebase, LRPing constantly with it and massive headaches in terms of moderating/administrating split personality has been disabled. 🦀 Split personality is gone 🦀. Also statistically its possible to roll into a catgirl handhold tailtwine neon green hair and neon green athlete shorts ERP when you may not be into that stuff and since it is statistically possible it is bound to happen. Nothing wrong if you are into catgirls or anything listed, its just a silly example.
/🆑

### CODE REVIEW

**Comment by ShadeAware** [comment]
image

**Comment by IrkallaEpsilon** [comment]
Give me a sec or a few. Man "*coding*" is hard. Aka removing shit in this particular instance :').

Yeah okay I am very tired to be honest and should have done this in the morning.

Cant even remove shit right right now :(

**Comment by IrkallaEpsilon** [comment]
Oh my fucking god, they added an additional thing, that fucking spell is saved elsewhere, hold on. 

Lets see if it works

**Comment by IrkallaEpsilon** [comment]
Shade you are a fucking god, I wouldnt have found that. I am serious here, I currently have the perception of a lemon

File: code/datums/brain_damage/split_personality.dm [hunk 1]
```diff
@@ -1,3 +1,5 @@
+//Skyrats Edit Begin Removes Split Personality
```
Comment (Azarak) [inline code/datums/brain_damage/split_personality.dm]:
```suggestion
//SKYRAT EDIT REMOVAL BEGIN
```

File: code/datums/brain_damage/split_personality.dm [hunk 2]
```diff
@@ -235,3 +237,5 @@
 
 #undef OWNER
 #undef STRANGER
+*/
+// Skyrats Edit End Remove Split Personality
```
Comment (Azarak) [inline code/datums/brain_damage/split_personality.dm]:
```suggestion
//SKYRAT EDIT REMOVAL END
```

File: code/modules/spells/spell_types/personality_commune.dm [hunk 3]
```diff
@@ -1,3 +1,5 @@
+//Skyrats edit begin Removes Split personality
```
Comment (Azarak) [inline code/modules/spells/spell_types/personality_commune.dm]:
```suggestion
//SKYRAT EDIT REMOVAL BEGIN
```

File: code/modules/spells/spell_types/personality_commune.dm [hunk 4]
```diff
@@ -30,3 +32,5 @@
 		if(!isobserver(ded))
 			continue
 		to_chat(ded, "[FOLLOW_LINK(ded, user)] <span class='boldnotice'>[user] [name]:</span> <span class='notice'>\"[msg]\" to</span> <span class='name'>[trauma]</span>")
+*/
+//Skyrats edit end
```
Comment (Azarak) [inline code/modules/spells/spell_types/personality_commune.dm]:
```suggestion
//SKYRAT EDIT REMOVAL END
```

Comment (IrkallaEpsilon) [reply code/modules/spells/spell_types/personality_commune.dm]: Done daddy :eyes:

**Review by Azarak — APPROVED** [review]

**Comment by Azarak** [comment]
I can see this coming back as a more controlled quirk

**Pull request merged**